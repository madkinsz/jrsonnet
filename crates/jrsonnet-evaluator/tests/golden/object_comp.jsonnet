{ local t = 'a', ['h' + i + '_' + z]: if 'h' + (i - 1) + '_' + z in self then t + 1 else 0 + t for i in [1, 2, 3] for z in [2, 3, 4] if z != i }
