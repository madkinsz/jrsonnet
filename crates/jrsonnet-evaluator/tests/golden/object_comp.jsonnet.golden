{
   "h1_2": "0a",
   "h1_3": "0a",
   "h1_4": "0a",
   "h2_3": "a1",
   "h2_4": "a1",
   "h3_2": "0a",
   "h3_4": "a1"
}